kDriveCore/Data/UploadQueue/Queue/UploadQueue+Queue.swift

```diff
@@ -168,6 +173,9 @@ extension UploadQueue: UploadQueueable {
                             addToQueue: Bool = true) -> UploadOperationable? {
         let expiringActivity = ExpiringActivity()
         expiringActivity.start()
+        defer {
+            expiringActivity.endAll()
```

Comment:
This fixes the APIV3 branch as a case was not releasing it. a `defer` is more future proof.